glib/src/source.rs

```diff
@@ -805,9 +808,10 @@ where
 /// This function panics if called from a different thread than the one that
 /// owns the main context.
 #[doc(alias = "g_unix_fd_add_full")]
-pub fn unix_fd_add_local<F>(fd: RawFd, condition: IOCondition, func: F) -> SourceId
+pub fn unix_fd_add_local<D, F>(fd: D, condition: IOCondition, func: F) -> SourceId
 where
-    F: FnMut(RawFd, IOCondition) -> Continue + 'static,
+    D: AsFd,
```

Comment:
`impl AsFd`, etc

Reply:
Did you mean to use `x: impl AsFd` rather than `x: D` (with `where D: AsFd`)?

Reply:
Yes